### ISSUE TITLE
[PLAYBACK] Windows Fullscreen playback causes flickering

### ISSUE THREAD

**Issue by Quackdoc** [olive-editor/olive#2230]
**Commit Hash** 
7fa69ea

**Platform** 
Windows 11, WIndows 10

**Summary**
On fullscreen playback The window flickers as if fullscreen is being toggled off and on.

**Additional Information / Output**
Currently confirmed on RX580 and RX 5700 XT, Issue did not occur when using mesa's opengl->d3d12. GLFlush/GLFinish doesn't change anything either

**Comment by Quackdoc** [comment]
I had a friend test a GTX 970 and it seems like it has no issues, seems like it's yet another AMD bug

**Comment by ThomasWilshaw** [comment]
yes i'm not able to reproduce this on WIndows 10 with  NVIDIA card so you're probably correct
